pkg/controllermanager/controller/shoot/shoot_hibernation_control.go

```diff
@@ -16,19 +16,23 @@ package shoot
 
 import (
 	"context"
-	"fmt"
 	"reflect"
+	"sort"
 	"time"
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	gardenlogger "github.com/gardener/gardener/pkg/logger"
-	"github.com/sirupsen/logrus"
+	"github.com/go-logr/logr"
+	"github.com/robfig/cron"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Generally, it would be nice to group the imports as usual